Description: Make debos work on ipv6-only machines
Author: Helmut Grohne <helmut@subdivi.de>
Forwarded: no

When running debos on an ipv6-only machine, it sets up a virtual machine using
fakemachine and tries to access Debian repository. It first tries connecting
via IPv6, but notices that the fakemachine does not have any IPv6 addresses nor
routes so it proceeds to using IPv4. It sends an IPv4 package and qemu's user
network stack tries to send it on the host, but there it notices that the
IPv6-only host does not have any IPv4 addresses nor routes and likewise reports
that the network is unreachable. As a result, running a fakemachine on an
IPv6-only host results in the inability to access network resources.

--- golang-github-go-debos-fakemachine-0.0.9.orig/machine.go
+++ golang-github-go-debos-fakemachine-0.0.9/machine.go
@@ -299,9 +299,8 @@ Type=ether
 
 [Network]
 DHCP=ipv4
-# Disable link-local address to speedup boot
-LinkLocalAddressing=no
-IPv6AcceptRA=no
+LinkLocalAddressing=yes
+IPv6AcceptRA=yes
 `
 
 const networkdLinkTemplate = `
